content/applications/finance/accounting/bank/feeds/bank_synchronization.rst

```diff
@@ -95,14 +91,14 @@ Synchronization in error
 ------------------------
 
 To report a connection error to the `Odoo support <https://www.odoo.com/help>`_, go to
-:menuselection:`Accounting --> Configuration --> Online Synchronization`, select the connection
+:menuselection:`Configuration --> Online Synchronization`, select the connection
 that failed, and copy the error description and the reference.
 
 Synchronization disconnected
 ----------------------------
 
 If your connection with the proxy is disconnected, you can reconnect with the proxy using the
-*Reconnect* button.
+:guilabel:`fetch account` button.
```

Comment:
Try to be consistent: 'fetch account' or 'Fetch Account' button, not both :) I also have seen 'Fetch Account**s**'